### ISSUE TITLE
Dark mode about page link colors are hard to see

### ISSUE THREAD

**Issue by yuanLeeMidori** [Seneca-CDOT/telescope#2048]
**What happened**:
![image](https://user-images.githubusercontent.com/50813726/112924996-1d61fd80-90df-11eb-98ea-6d3b73a91e33.png)

**What should have happened**:
The visited link and unvisited link font color should be updated.
**How to reproduce it (as precise as possible)**:
I'd like to add more color to dark mode palette and update the color in dark mode about page. (I think the link colors in dark mode post can be reused here)
**Anything else we need to know?**:

**Environment**:

- OS:
- Browser:
